source/includes/_subscriptions.md

```diff
@@ -439,3 +439,101 @@ size | `number` | Number of subscriptions per page _default is 10_
 page | `number` | Which page to retrieve _default is 0_
 sort | `string` | Field used to sort results _default is `-modified`_
 status | `string` | The subscription status _default is `ACTIVE`_
+
+
+## Recurring Order Subscriptions Object
+
+Create this type of plan when you need to schedule some [`Orders`](#Orders) daily, weekly, or monthly.
+To plan a schedule follow the next instruction on [`Recurring Order Plan](#recurring-order-plan).
+
+Attribute | Type | Description
+--------- | ---- | -------
+_id | `object` | The subscription ID.
+method | `string` | Name of subscription method. `recurring_order`
+name | `string` | The name of the subscription type. _default is `RECURRING_ORDER`_
+note | `string` | A short description of the description.
+user | `object`  | [`User`](#Users) ID associated with the subscription.
+status | `string` | Status for the subscription. _default is `CREATED`, also available are `ACTIVE`, `FUTURE`, `NON_RENEWING`and `CANCELLED`_
+plan | `object` | ID of the [`Recurring Order Plan`](#recurring-order-plan) associated with the subscription.
+payment_method | `object` | ID of already created payment method.
+payments | `array` | Array of [`Payment`](#payments) objects associated with the subscription.
+starting_date | `string`| Start date, `timestamp` format. _default is current date if missing_ 
+ending_date | `string`| End date of subscription, `timestamp` format .
+interval_total | `number` | Number of intervals set in the [`Plan`](#Plans) the subscription will run.
+infinite | `boolean` | If `true`, the subscription will run until the user stops it.
+current_billing_date_period_start | `string`| The date on which the customer was billed last.
+current_billing_date_period_end | `string`| The date on which the customer will be billed next. This will also be the date on which the next current_billing_date_period_start of the subscription starts.
+prorate_amount | `number` | The amount prorated. Currency set in [`Plan`](#Plans).
+prorate_date | `number`| The date of the last time an amount was prorated
+last_billing_amount | `number`| The amount that was subtracted at the last payment. `plan.total_amount`+`prorate_amount`
+total_fail_attempts | `number` | The number of failed payment attempts in the subscription. See [`Plan`](#Plans)`.max_fail_attempts
+initial_order | `object` | [`Order`](#Orders) ID associated with the subscription and used to create a [`Recurring Order Plan`](#recurring-order-plan)
```

Comment:
Not ID...

The initial [`order`](#Orders) in the subscription. It is used to create a [`Recurring Order Plan`](#recurring-order-plan)